### ISSUE TITLE
Could you provide a data layer demo to read image and corresponding groundtruth mask?

### ISSUE THREAD

**Issue by JunMa11** [zizhaozhang/unet-tensorflow-keras#1]
Excuse me, I'm a fresh man in tensorflow and keras. I failed to create a data layer demo.
Would it be possible for you to provide a data layer demo to read image and corresponding groundtruth mask ?
I would appreciate your help.

**Comment by zizhaozhang** [comment]
I like to use Keras to load image with mask and do automatic augmentation online.
Please see this example, https://keras.io/preprocessing/image/. Very straightforward
